usagov/vote-gov
# VOTE-246 Integrate nvrf app into static site

**Testing steps**
1. run `npm i`
2. run `npm run build`
3. run `npm start`
4. visit `/nvrf-test` and see the NVRF app embedded on the site.
